transformer_lens/HookedTransformer.py

```diff
@@ -513,6 +520,12 @@ def to_single_token(self, string):
         assert not token.shape, f"Input string: {string} is not a single token!"
         return token.item()
 
+    def to_single_str_token(self, int_token: int) -> str:
+        # Gives the single token corresponding to an int, in string form
+        assert isinstance(int_token, int)
+        token = self.to_str_tokens(torch.tensor([int_token]))
+        return token[0]
```

Comment:
Nit: Maybe this should be `token.item()`? `item` returns a single int/float if the tensor has a single element, otherwise raises an error. Here they *should* do the same thing, but it's a good sanity check that you aren't silently throwing away some extra stuff (eg, if you have a dumb tokenizer that somehow maps a single element tensor of tokens to multiple strings.